src/reducers/entities/integrations.js

```diff
@@ -122,6 +127,34 @@ function commands(state = {}, action) {
     }
 }
 
+function executableCommands(state = {}, action) {
```

Comment:
I'm wondering if this would be best to call it `builtInCommands` instead of `executableCommands` but I'm 0/5, definitely non-blocking for me

Reply:
Maybe even `systemCommands` because really they're just commands without teams right? And "system" kind of means "built-in" too

Reply:
I'm good with systemCommands. @enahum work for you?

Reply:
I don't know if you saw my reaction, but just in case the answer was YES